pavelib/assets.py

```diff
@@ -46,17 +47,25 @@
 # A list of NPM installed libraries that should be copied into the common
 # static directory.
 NPM_INSTALLED_LIBRARIES = [
-    'jquery/dist/jquery.js',
-    'jquery-migrate/dist/jquery-migrate.js',
-    'jquery.scrollto/jquery.scrollTo.js',
-    'underscore/underscore.js',
-    'underscore.string/dist/underscore.string.js',
-    'picturefill/dist/picturefill.js',
+    'backbone-validation/dist/backbone-validation-min.js',
     'backbone/backbone.js',
     'edx-ui-toolkit/node_modules/backbone.paginator/lib/backbone.paginator.js',
-    'backbone-validation/dist/backbone-validation-min.js',
     'edx-ui-toolkit/node_modules/moment-timezone/builds/moment-timezone-with-data.js',
     'edx-ui-toolkit/node_modules/moment/min/moment-with-locales.js',
+    'jquery-migrate/dist/jquery-migrate.js',
```

Comment:
In general, I'm not clear what needs to be listed here. requirejs-text is also not listed.

And what about backbone.paginator, which is being installed via npm but also on line 52?